src/shared/cloudformation/activation.ts

```diff
@@ -93,6 +103,9 @@ function addCustomTags(): void {
         '!Split',
         '!Split sequence',
     ]
-    const updateTags = currentTags.concat(cloudFormationTags.filter(item => currentTags.indexOf(item) < 0))
-    vscode.workspace.getConfiguration().update('yaml.customTags', updateTags, vscode.ConfigurationTarget.Workspace)
+    const missingTags = cloudFormationTags.filter(item => !currentTags.includes(item))
+    if (missingTags.length > 0) {
+        const updateTags = currentTags.concat(missingTags)
+        vscode.workspace.getConfiguration().update('yaml.customTags', updateTags, vscode.ConfigurationTarget.Global)
```

Comment:
Write to Global scope. Also don't update settings if the setting already contains the symbols exist...